src/lib/prof/NameMappings.cpp

```diff
@@ -47,6 +47,7 @@ const char *GPU_COPYIN  = "<gpu copyin>";
 const char *GPU_COPYOUT = "<gpu copyout>";
 const char *GPU_ALLOC   = "<gpu alloc>";
 const char *GPU_DELETE  = "<gpu delete>";
+const char *GPU_MEMSET  = "<gpu memset>";
```

Comment:
@jmellorcrummey This is a top priority fix. Without adding the missing tag, hpcrun generates some weird call paths on summit. Then, hpcprof is confused and failed to identify the GPU kernel launch site. As a result, no GPU calling context tree will be reconstructed. 

This fix solves the above problem.

Reply:
Update...

This does not seem to be the only culprit. There are other problems I am still investigating.